runtime/grid_reduction.cu

```diff
@@ -857,6 +878,30 @@ __device__ void iterGroupedGridReduce(
         index_utils::maskedOffset<!X_THREAD, !Y_THREAD, !Z_THREAD>(
             threadIdx, blockDim);
 
+    // Max vectorized load/store size is 16 bytes, if each thread has more than
```

Comment:
Let's make a constexpr function calculate total_bytes, n_loads and n_elements. They are the same here and the lastblock version, right?

Reply:
yes, added two constexpr functions to calculate n_loads and n_elements seperatelly. I can't directly use std::pair it is not in our runtime.